compiler/rustc_resolve/src/check_unused.rs

```diff
@@ -27,14 +27,16 @@ use crate::imports::ImportKind;
 use crate::module_to_string;
 use crate::Resolver;
 
-use crate::NameBindingKind;
+use crate::{LexicalScopeBinding, NameBindingKind};
 use rustc_ast as ast;
 use rustc_ast::visit::{self, Visitor};
 use rustc_data_structures::fx::{FxHashMap, FxIndexMap, FxIndexSet};
 use rustc_data_structures::unord::UnordSet;
 use rustc_errors::{pluralize, MultiSpan};
 use rustc_hir::def::{DefKind, Res};
-use rustc_session::lint::builtin::{MACRO_USE_EXTERN_CRATE, UNUSED_EXTERN_CRATES, UNUSED_IMPORTS};
+use rustc_session::lint::builtin::{
+    MACRO_USE_EXTERN_CRATE, UNUSED_EXTERN_CRATES, UNUSED_IMPORTS, UNUSED_QUALIFICATIONS,
+};
```

Comment:
Style nit: could you keep the single line import style?

Reply:
Thank you. I tried removing the comma at the end and putting on one line, but `x.py fmt`  format it like this.

Reply:
> Style nit: could you keep the single line import style?

Thank you. I tried removing the comma at the end and putting on one line, but x.py fmt format it like this.

Reply:
I meant splitting imports into multiple lines if they are too long for one line.
But that's not too important.